roles/kafka_connect/tasks/main.yml

```diff
@@ -4,6 +4,13 @@
   when: not common_role_completed|bool
   tags: common
 
+- name: Check z/OS Preconditions
+  fail:
+    msg: 'z/OS must be installed from archive'
```

Comment:
Should I read it as "On z/OS, Connect service can be installed via archive"

Reply:
Yeah, I'll change it to "On z/OS Confluent Platform Services MUST be installed via archive"